types/xlsx/index.d.ts

```diff
@@ -173,13 +173,147 @@ export interface IWorkBook {
     Props: IProperties;
 }
 
+export interface IColInfo {
+    /**
+     * Excel's "Max Digit Width" unit, always integral
+     */
+    MDW?: number;
+    /**
+     * width in Excel's "Max Digit Width", width*256 is integral
+     */
+    width: number;
```

Comment:
This is copied from the [Column Properties](https://sheetjs.gitbooks.io/docs/#column-properties) section of the SheetJS docs; it sounds like that's also incorrect? (I did have to correct the typo of `hidden:?boolean` so I wouldn't be surprised.)